### PR TITLE
Belarusian: fix IPA chars

### PR DESCRIPTION
**Pull request by bespsm**
- fix IPA chars for "s;" and "z;" phonemes
- add bug note about "...i a..." case

### CODE REVIEW

File: dictsource/be_rules [hunk 1]
```diff
@@ -80,7 +80,7 @@
        з (L06L04 z;
        з (ці   s;
 
-.group і
+.group і       // NOTE: appends ";" at the end of the words when next starts from "а"
```
Comment (HaraldZealot) [inline dictsource/be_rules:83]: I'm not getting this comment

**Review by HaraldZealot — APPROVED** [review]
LGTM

File: dictsource/be_rules (hunk 1 above)
Comment (bespsm) [reply dictsource/be_rules:83]: @HaraldZealot  check this phonemes, "суткі":
https://github.com/espeak-ng/espeak-ng/blob/master/tests/language-pronunciation.test#L25

**Comment by bespsm** [comment]
seems like there is an issue with macos compilers.
bu please review, @kirill-jjj @alex19EP @jaacoppi

**Pull request merged**